### PR TITLE
preliminary birdlife threats

### PR DESCRIPTION
**Pull request by davharris**
Looks like I got most of the interesting material out of the threats table.  If this looks good, I can write up documentation.

Example usage:

```
birdlife_threats(22721692) # Has listed threats, returns data.frame
birdlife_threats(22705990) # Has no listed threats, returns NULL
```

### CODE REVIEW

File: R/birdlife.R [hunk 1]
```diff
@@ -25,3 +25,46 @@ birdlife_habitat = function(id){
   out = cbind(id, tables[[habitat_table_number]])
   out[-nrow(out), ] # Drop last row (altitude)
 }
+
+
```
Comment (sckott) [inline R/birdlife.R]: @davharris don't we need an `#' @export` here - and a `#' @rdname birdlife_habitat`, or if you want to give it a separate man file, then that stuff

Comment (davharris) [inline R/birdlife.R]: Ok.  I wasn't going to bother with roxygen until it was reviewed, but I'll do that now.

Comment (sckott) [inline R/birdlife.R]: Cool, need to associate it with a man file though...currently, nothing happens when you execute `?birdlife_threats`, maybe #' @rdname birdlife_habitat

**Comment by sckott** [comment]
@davharris  Could you include some egs in the man file?

File: R/birdlife.R (hunk 1 above)
Comment (davharris) [inline R/birdlife.R]: Sorry, my intent was to put up the code and then add documentation once it got the thumbs up.  I'll make sure not to push prematurely next time.

Comment (sckott) [inline R/birdlife.R]: no worries - just wondering if you forgot :+1:

**Comment by davharris** [comment]
Should the examples be commented out so that CRAN doesn't ping birdlife?

**Comment by sckott** [comment]
yes, in `\dontrun` plz

**Comment by davharris** [comment]
Ok, this might be ready to merge now.

**Comment by sckott** [comment]
Looks good!

**Pull request merged**